contrib/babelfishpg_tds/src/backend/tds/tdsresponse.c

```diff
@@ -2759,6 +2758,14 @@ TdsPrintTup(TupleTableSlot *slot, DestReceiver *self)
 						 */
 						TdsPutInt16LE(0xffff);
 						break;
+					case TDS_TYPE_XML:
+
+						/*
+						 * In case of TDS version lower than or equal to 7.3A,
+						 * we need to send 0xffffffffffffffff (PLP_NULL)
+						 */
+						TdsPutUInt64LE(0xffffffffffffffff);
+						break;
```

Comment:
Code where we calculate simpleRowSize needs to be updated according to this. And do we have existing test cases for this? if so please put a reference here and if not then plz add some

Reply:
There are no existing tests for this case, adding them in `TestXML` test file